SwissTransferCore/Settings/SettingReader.swift

```diff
@@ -32,6 +32,10 @@ public final class NotificationSettings {
     ]
 
     public var allEnabled: Bool {
+        guard let firstKey = keys.first,
```

Comment:
Isn't it a bit hacky?

Could we imagine something like kMail, with variables to access each key. These properties return the value or the DefaultValue if nil.
Then, the array of keys could contains KeyPath to these properties, so you can check the value

Reply:
I might not be clear, feel free to ask me ;)

Reply:
Yeah it was a quickfix. Like we discussed, I matched what was made on the _mail_ app to set some default values.

I also think that this can be cleaner not repeating code to set default for each and every value using the `func register(defaults registrationDictionary: [[String](https://developer.apple.com/documentation/swift/string) : Any])` function. I created a ticket to discuss next monday.